pkgs/tools/typesetting/tex/texlive/combine.nix

```diff
@@ -17,6 +17,14 @@ let
   pkgList = rec {
     combined = combinePkgs (lib.attrValues pkgSet);
     all = lib.filter pkgFilter combined;
+    licenses = lib.foldl (acc: pkg:
```

Comment:
Maybe `mergeLicenses`, or `concatLicenses`? I find the double fold hard to read, but as long as the performance is reasonable, it seems ok.